config.go

```diff
@@ -2,3 +2,5 @@ package main
 
 // CtaAPIKey is the API key required to access the HTTP API from CTA
 const CtaAPIKey = ""
+const serverHostname = ""
```

Comment:
I'd suggest to make the API key, hostname and port more customizable, e.g. by passing it via args or env variables